cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/GetBrokerStatsRunnable.java

```diff
@@ -41,7 +43,8 @@ protected BrokerStats getResult() throws Exception {
     if (cachedBrokerStats != null) {
       return cachedBrokerStats;
     }
-    return _kafkaCruiseControl.clusterModel(_time,
+    return _kafkaCruiseControl.clusterModel(_start,
```

Comment:
In case the `start` is `-1`, should we call `clusterModel(long, ModelCompletenessRequirements, OperationProgress)`? This ensures that the `_cachedBrokerLoadStats` and `_cachedBrokerLoadGeneration` are updated in typical use-cases.